packages/core/src/contents/client/__fixtures__/contents.fixtures.js

```diff
@@ -22,6 +23,7 @@ export default {
       moxios.stubRequest(
         join('/api/content/v1/search/contents', {
           query: params.queryParams,
+          queryOptions: { encode: false },
```

Comment:
Do you really need this prop?

Reply:
Yes! Without this the comma (,) on the target.segments will be encode to hex